### ISSUE TITLE
Can't use client.decode_jwt.

### ISSUE THREAD

**Issue by daikidomon** [paypay/paypayopa-sdk-python#56]
**Describe the bug**

Unable to get userAuthorizationId from ResponseToken with the following error.

```
JWT Signature verification failed:  It is required that you pass in a value for the "algorithms" argument when calling decode().
```

**To Reproduce**
```python
client = paypayopa.Client(auth=(API_KEY, API_SECRET),
                          production_mode=False)
client.decode_jwt(secret=API_SECRET, token=token)
```

**Expected behavior**
Decoding "responseToken", and get UserAuthorizationID.

**Screenshots**
Nothing

**Additional context**
A responseToken is "eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJyZXN1bHQiOiJzdWNjZWVkZWQiLCJhdWQiOiJhX2hTNFpNdUw4RUciLCJpc3MiOiJwYXlwYXkubmUuanAiLCJwcm9maWxlSWRlbnRpZmllciI6IioqKioqKiowMDc3IiwiZXhwIjoxNjEyMTk2MDU2LCJub25jZSI6Imtpa2FrdSIsInVzZXJBdXRob3JpemF0aW9uSWQiOiIzNjc4ZjYzNS03N2VkLTQ5NzItOWFhZS1mZjU0MjA4NTU5ZTQiLCJyZWZlcmVuY2VJZCI6IjAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDQifQ.S_iSNEmI6kGPop9LQwqISIF-bwepG7AD1bVhwyWg16M".

**Comment by shreyanshp** [comment]
@solancer Please check this and try to fix it before tomorrow's release

**Comment by daikidomon** [comment]
The following is a link that may be helpful.
https://pyjwt.readthedocs.io/en/stable/usage.html#encoding-decoding-tokens-with-hs256

And, I guess we need to change the way we specify the args of "options" .

**Comment by shreyanshp** [comment]
Released in v1.0.1
